src/flamegraph/merge.rs

```diff
@@ -125,9 +125,9 @@ where
         let nsamples = if let Some(s2) = parse_nsamples(&mut line) {
```

Comment:
I wonder if we can give this a more descriptive name than `s2`?

Reply:
How about we change `s2` to just `samples`. Then if there's a differential column, we can use something like `samples_first` instead of `s1`. Or do you have a better suggestion?

Reply:
I like that! Maybe `original_samples` for `s1`?

Reply:
Sure